tests/integrated/test-delp2/runtest

```diff
@@ -21,8 +21,9 @@ shell_safe("make > make.log")
 exefile = "./test_delp2"
 
 # List of settings to apply
-settings = ["mxg=2 mesh:nx=36", 
-            "mxg=1 mesh:nx=34"]
+settings = ["mxg=2 mesh:nx=36 diffusion:useFFT=true", 
+            "mxg=1 mesh:nx=34 diffusion:useFFT=true", "mxg=2 mesh:nx=36 diffusion:useFFT=false", 
+            "mxg=1 mesh:nx=34 diffusion:useFFT=false"]
 
 
 success = True
```

Comment:
I'm confused. It looks like `test-delp2` just compares the result from 1 processor against the results from 2 and 4 processors. Am I reading it right? If I am it doesn't seem like a very good test, I'd think it would be better to save some benchmark results for regression testing.

Reply:
I didn't look beyond the settings line! I'd prefer to avoid storing binary benchmark data if possible but agree that it would be better to test the output against something other than itself.